src/neptune/internal/utils/logger.py

```diff
@@ -49,12 +49,11 @@ def stream(self):
         return sys.stdout
 
 
-def get_logger(with_prefix: bool = True, loglevel: int = logging.DEBUG):
+def get_logger(with_prefix: bool = True):
     if with_prefix:
         logger = logging.getLogger(NEPTUNE_LOGGER_NAME)
     else:
         logger = logging.getLogger(NEPTUNE_NO_PREFIX_LOGGER_NAME)
-    logger.setLevel(loglevel)
```

Comment:
What level are we going to have now? Maybe this should be just a part of `_set_up_logging` and `_set_up_no_prefix_logging`?

Reply:
Debug it's for rather developers. `INFO` should be ok.